packages/checkbox/src/Checkbox.ts

```diff
@@ -125,11 +125,12 @@ export class Checkbox extends SizedMixin(CheckboxBase) {
                 this.inputElement.removeAttribute('aria-invalid');
             }
         }
-        if (changes.has('indeterminate')) {
+        if (changes.has('indeterminate') || this.indeterminate) {
```

Comment:
There's further question in https://github.com/adobe/spectrum-web-components/pull/3267#discussion_r1214795811 before we can move forward with this...